### PR TITLE
Ran prettier on SASS files

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Now that the low-hanging fruit PR is done, #5890, for the homepage makeover, we can format all the SCSS files via prettier.

NOTE: We have a precommit task that runs prettier on all *.scss files.

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [x] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

A pretty dandelion

### CODE REVIEW

**Comment by nickytonline** [comment]
Grab this locally @ludwiczakpawel, but it should all be good.

File: app/assets/stylesheets/article-show.scss [hunk 1]
```diff
@@ -395,20 +375,11 @@ article {
...
     width: 82%;
-    font-family:
-      Palatino,
-      'Palatino Linotype',
-      'Palatino LT STD',
-      'Book Antiqua',
-      Georgia,
-      serif;
+    font-family: Palatino, 'Palatino Linotype', 'Palatino LT STD',
+      'Book Antiqua', Georgia, serif;
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/article-show.scss]: Why is this split into two lines?

File: app/assets/stylesheets/article-show.scss [hunk 2]
```diff
@@ -1198,7 +1138,7 @@ article {
             margin-left: 15px;
           }
 
-          input[type="checkbox"] {
+          input[type='checkbox'] {
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/article-show.scss]: Any particular reason for using `'` over `"` ? Asking out of curiosity..

File: app/assets/stylesheets/credits.scss [hunk 3]
```diff
@@ -117,8 +107,16 @@
     @include themeable(color, theme-secondary-color, $medium-gray);
     padding: 25px 12px;
     font-size: 0.8em;
-    @include themeable(border-top, theme-container-border, 1px solid lighten($light-medium-gray, 7%));
-    @include themeable(border-bottom, theme-container-border, 1px solid lighten($light-medium-gray, 7%));
+    @include themeable(
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/credits.scss]: Why is this sometimes split into multiple lines and sometimes it's the other way: https://github.com/thepracticaldev/dev.to/pull/6207/files#diff-9f426ab011d6177be0c850da788ef089R49 ?

File: app/assets/stylesheets/article-show.scss (hunk 1 above)
Comment (nickytonline) [reply app/assets/stylesheets/article-show.scss]: It’s just prettier formatting. There is a max line width

File: app/assets/stylesheets/article-show.scss (hunk 2 above)
Comment (nickytonline) [reply app/assets/stylesheets/article-show.scss]: Once again prettier formatting. Pull it locally to try the branch.

**Comment by nickytonline** [comment]
> I think it's fine to merge this, but this is a case for PR review apps. That would make it easier for @lisasy and @ludwiczakpawel to poke around without having to pull locally.

I just said to pull locally as @ludwiczakpawel was asking about all the formatting, but the app still behaves the same.

As @rhymes said up top, trust the tools. 😉

**Pull request merged**